lib/chroot-buildpackages.sh

```diff
@@ -86,9 +103,20 @@ create_chroot()
 		mkdir -p "${target_dir}"/var/lock
 	fi
 	chroot "${target_dir}" /bin/bash -c "/usr/sbin/update-ccache-symlinks"
-	[[ $release == bullseye || $release == focal || $release == hirsute || $release == sid ]] && chroot "${target_dir}" /bin/bash -c "ln -s /usr/bin/python3 /usr/bin/python"
+
+	case $release in
+		bullseye|focal|hirsute|sid)
+			chroot "${target_dir}" /bin/bash -c "ln -s /usr/bin/python3 /usr/bin/python"
```

Comment:
Using this package would be a more appropriate solution

https://packages.debian.org/sid/main/python-is-python3

Reply:
There is no doubt that this is the best solution.
Thanks.